packages/compiler/src/server/completion.ts

```diff
@@ -237,6 +246,39 @@ async function addRelativePathCompletion(
   }
 }
 
+function addModelCompletion(context: CompletionContext, posDetail: PositionDetail) {
+  const node = posDetail.node;
+  if (
+    node.kind !== SyntaxKind.ModelStatement &&
+    node.kind !== SyntaxKind.ModelExpression &&
+    node.kind !== SyntaxKind.ObjectLiteral
+  ) {
+    return;
+  }
+  // skip the scenario like `{ ... }|`
+  if (node.end === posDetail.position && posDetail.preChar === CharCode.CloseBrace) {
```

Comment:
here what I was more thinking is checking if you are not part of the model node(checking `end` property) instead of checking any syntax in here which could break.

Reply:
remove the check of close brace